duckduckgo/BrowserServicesKit
# Update Autofill submodule pointer with iOS fix

Task/Issue URL: https://app.asana.com/0/0/1201729396487492/f
Tech Design URL:
CC: @GioSensation 

**Description**:

This PR updates the Autofill submodule pointer to point to the most recent changes which restore Autofill functionality on iOS.

**Steps to test this PR**:
1. In the iOS repo, check out [sam/update-bsk-to-release-branch](https://github.com/duckduckgo/iOS/tree/sam/update-bsk-to-release-branch) and test the Autofill functionality
1. In the macOS repo, check out [sam/update-bsk-with-autofill-ios-fix](https://github.com/more-duckduckgo-org/macos-browser/tree/sam/update-bsk-with-autofill-ios-fix) and test the Autofill functionality



**OS Testing**:


---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
